### PR TITLE
feat(dashboards): Display an alert banner in Discover when opening from a MEPs Widget

### PR DESCRIPTION
**Pull request by edwardgou-sentry**
Adds an alert banner when a user visits a Discover Result page with `fromMetric=true`.
`fromMetric=true` not used anywhere yet though.
image

### CODE REVIEW

File: static/app/views/eventsV2/results.tsx [hunk 1]
```diff
@@ -121,6 +122,13 @@ class Results extends Component<Props, State> {
 
   componentDidMount() {
     const {organization, selection, location} = this.props;
+    if (location.query.fromMetric) {
+      this.setState({showMetricsAlert: true});
```
Comment (shruthilayaj) [inline static/app/views/eventsV2/results.tsx:126]: should we clear the alert when the user modifies the query?

**Review by shruthilayaj — APPROVED** [review]

File: static/app/views/eventsV2/results.tsx [hunk 2]
```diff
@@ -468,6 +476,19 @@ class Results extends Component<Props, State> {
     this.setState({error, errorCode});
   };
 
+  renderMetricsFallbackBanner() {
+    if (this.state.showMetricsAlert) {
+      return (
+        <Alert type="info" showIcon>
+          {
+            "You've navigated to this page from an unsampled query. The results displayed here only show sampled data, which may differ from the original unsampled query."
```
Comment (shruthilayaj) [inline static/app/views/eventsV2/results.tsx:484]: localization
```suggestion
            t("You've navigated to this page from an unsampled query. The results displayed here only show sampled data, which may differ from the original unsampled query.")
```

Comment (edwardgou-sentry) [reply static/app/views/eventsV2/results.tsx]: 👍

File: static/app/views/eventsV2/results.tsx (hunk 1 above)
Comment (edwardgou-sentry) [reply static/app/views/eventsV2/results.tsx:126]: Good question, I'll look into it and we can add an another pr!

**Pull request merged**